core/object/script_language_extension.h

```diff
@@ -387,6 +387,8 @@ class ScriptLanguageExtension : public ScriptLanguage {
 				option.icon = op["icon"];
 				ERR_CONTINUE(!op.has("default_value"));
 				option.default_value = op["default_value"];
+				ERR_CONTINUE(!op.has("location"));
+				option.default_value = op["location"];
```

Comment:
I was not sure if here you would prefer location to be optional:
```cpp
if (op.has("location)){
    option.location = op["location'];
}
```
So I just followed what was already there.